next.json

```diff
@@ -280,6 +280,13 @@
                 ]
             },
             {
+                "name": "set stencil reference",
+                "args": [
+                    {"name": "back reference", "type": "uint32_t"},
```

Comment:
Also it would be nice to define a default value for this in a renderpass, so that we don't have UB when a draw needs to use the reference and it hasn't been set in the command buffer before.